duckduckgo/iOS
# Deprecating harmful terminology

Task/Issue URL: https://app.asana.com/0/856498667320406/1180448683138619
Tech Design URL:
CC:

**Description**:
UI and code changes aimed at deprecating harmful terminology. We know some users don't know what "whitelist" means, so clearer terminology might also improve usability and comprehension.
**Steps to test this PR**:
1. Ensure Privacy Protection toggle works.
2. Check strings:
 - Settings.
 - Privacy protection.
 - Unprotected Sites list (including empty state and add new alert)
 - Menu (toggling protection should present toast).
3. Ensure other functionality works:
 - HTTPS upgrade (along with excluded sites).
 - HTTPS Upgrade Excluded Sites DB migration.
 - Refetching and Storing ETAGs for temporary Unprotected sites and sites excluded from HTTPS upgrades.



**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
